packages/core/src/shared/clients/devenvClient.ts

```diff
@@ -138,10 +136,8 @@ export class DevEnvActivity implements vscode.Disposable {
 
     static readonly activityUpdateDelay = 10_000
 
-    /**
-     * Returns an instance if the activity mechanism is confirmed to be working.
-     */
-    static async instanceIfActivityTrackingEnabled(
+    /** Gets a new DevEnvActivity, or undefined if service is failed. */
+    static async create(
```

Comment:
This is a case where the descriptiveness of the function name is confusing. This function is not doing anything unusual, but its name was unusual.